Aquality.Selenium/src/Aquality.Selenium/Browsers/Browser.cs

```diff
@@ -173,6 +182,15 @@ private INavigation Navigate()
             return new BrowserNavigation(Driver);
         }
 
+        /// <summary>
+        /// Provide interface to manage of browser tabs.
+        /// </summary>
+        /// <returns>instance of IBrowserTabNavigation.</returns>
+        public IBrowserTabNavigation TabNavigation()
```

Comment:
may we name it `public IBrowserTabNavigation Tabs()`?

Reply:
okay